src/Components/Shared/Inputs/DatePicker.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import styles from './inputs.module.css';
+
+const DatePicker = ({ value, onChangeDate }) => {
```

Comment:
should be great to have props for "name" and "label"

Reply:
👍